packages/compose-tunnel-agent/src/ssh/tunnel-client.ts

```diff
@@ -131,26 +125,35 @@ export const sshClient = async ({
     clientId, tunnels: tunnelsFromHelloResponse(tunnels),
   })
 
+  const getForwardRequestId = (service: {name: string; project: string; port: number; access: 'private' | 'public'}) => {
```

Comment:
reconstructing the request ID from the details is code duplication, plus it can be ambiguous if more key-value pairs are introduced. instead the original ID needs to be saved with the forward object in `tunnelSet`.

Reply:
you should view the usage in the file..
we need to construct the requestId from running services.. no much other way around it